nightwatch.conf.js

```diff
@@ -0,0 +1,18 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const testSettings = require('./packages/terra-toolkit/lib/index').testSettings;
```

Comment:
Talked with @Matt-Butler offline about this and since this is at the root level, it needs to have a path unfortunately.